cmd/eksctl-anywhere/cmd/supportbundle.go

```diff
@@ -89,7 +89,7 @@ func (csbo *createSupportBundleOptions) createBundle(ctx context.Context, since,
 	}
 
 	deps, err := dependencies.ForSpec(ctx, clusterSpec).
-		WithProvider(csbo.fileName, clusterSpec.Cluster, cc.skipIpCheck, csbo.hardwareFileName, false, csbo.tinkerbellBootstrapIP).
+		WithProvider(csbo.fileName, clusterSpec.Cluster, cc.skipIpCheck, csbo.hardwareFileName, false, csbo.tinkerbellBootstrapIP, map[string]bool{}).
```

Comment:
why does support bundle need this information?

Reply:
oh no it doesn't. Hence the empty object. Similar to the `false` being passed in. Its mainly because its a shared function.

Reply:
Got it. This feels a bit like a code smell to me if components that don't need a resource have to take it as an argument.

Not anyone's fault and definitely not something to fix for this PR, just something to think about.

Reply:
Yea i wasn't too happy with it either. I wanted to keep it as optional but didnt find a good way to pass along the map since go does it as array of options and i had only one specific thing to pass.